error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StreamRulesTestInput.java

```diff
@@ -322,4 +324,12 @@ Stream<Integer> testStreamOf4() {
   Stream<Integer> testStreamOf5() {
     return ImmutableList.of(1, 2, 3, 4, 5).stream();
   }
+
+  Stream<String> testStreamsStream() {
+    return StreamSupport.stream(((Iterable<String>) ImmutableList.of("foo")).spliterator(), false);
```

Comment:
Usually we use `Iterables.cycle` to create an `Iterable`-that-is-no-also a `Collection` :)